src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -31,17 +31,24 @@ const propTypes = {
 
     /** From Onyx */
     reimbursementAccount: reimbursementAccountPropTypes,
```

Comment:
I think we should add this here too, since we're adding comments about each individual onyx prop :D

```suggestion
    /** Bank account currently in setup */
    reimbursementAccount: reimbursementAccountPropTypes,
```